sites/public/src/components/account/StatusItem.tsx

```diff
@@ -3,39 +3,62 @@ import { t } from "@bloom-housing/ui-components"
 import { Button, Card, Tag } from "@bloom-housing/ui-seeds"
 import styles from "./StatusItem.module.scss"
 import applicationsViewStyles from "./ApplicationsView.module.scss"
+import { ListingsStatusEnum } from "@bloom-housing/shared-helpers/src/types/backend-swagger"
 
 interface StatusItemProps {
   applicationDueDate?: string
   applicationURL: string
-  applicationUpdatedAt: string
   confirmationNumber?: string
   listingName: string
   listingURL: string
+  listingStatus: ListingsStatusEnum
+  lotteryStartDate?: string
+  lotteryPublishedDate?: string
   lotteryResults?: boolean
   lotteryURL?: string
   strings?: {
```

Comment:
Is this still worth building out? Considered adding custom string options for open closed, lottery run, applications close, lottery date, and lottery published at strings but not really sure how this is applicable now if the component isn't shared. Any thoughts @emilyjablonski

Reply:
Yeah we used to export this, since we're just using it internally now we don't need this